src/EditorFeatures/CSharp/InlineRename/CSharpRenameContextHelper.cs

```diff
@@ -0,0 +1,636 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copied from VisualStudio.Conversations repo src/Copilot.Vsix/QuickActions/CSharp/CSharpTypeSignatureHelper.cs
+// with minor changes
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.InlineRename
+{
+    internal static class RenameContextHelper
```

Comment:
I don't see this class is used? 🤔

Reply:
I still don't see this class is used in the lastest commit🤔 Is this for any special purpose?

Reply:
Uncertain what all of this is for.

Reply:
Ah I see it provides the extension method `TryGetSurroundingNodeSpanAsync`